frontend/src/util/err.tsx

```diff
@@ -150,8 +148,14 @@ export const ErrorDisplay: React.FC<ErrorDisplayProps> = ({ failedAction, ...pro
         <p css={textColor}>
             {failedAction && failedAction + " "}
             {causes.length === 1
-                ? causes[0] + " "
-                : <ul>{causes.map(cause => <li key={cause}>{cause}</li>)}</ul>
+                // @ts-expect-error: Dynamically passed i18next keys need to be typed
+                // more strictly than just `string`. But I don't want to do that for all
+                // possible errors.
+                ? <Trans i18nKey={causes[0]} />
```

Comment:
That's not too bad. There is one type one can simply import that represents all valid translation keys. If only I remembered the name of the type :D I'm sure we use it somewhere already

Reply:
Dang yeah, I encounter the same error.  But this needs to be changed. Case in point: in `Upload.tsx` the `cases` array is also created, but that code is still calling `t`. So that is currently broken, as the already translated string will be used as translation key. And that's why proper typing is important, in particular `string` is way more dangerous than most assume.

But yeah ok how to fix it. I haven't played around with it a ton, but I couldn't get rid of this `too complex` error. Maybe just change `string` to `JSX.Element` or `React.Node` or sth? That way, the `Set` can simply contain anything that can be rendered. All places can keep calling `t` and just add strings to the set. But the one place that needs `Trans` due to `<br />` can just add `<Trans i18nextKey="..." />` to the set.